src/elements/Image.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+import { border, compose, layout, space } from "styled-system";
+
+const composedHelpers = compose(
+  layout,
+  border,
+  space
+);
+
+export const Img = styled.img`
+  width: 100%;
+  object-fit: cover;
+  ${composedHelpers};
+`;
+
+Img.defaultProps = {
+  // borderRadius: 8
```

Comment:
Kullanılmuyorsa yoruma çevirmek yerine silinebilir

Reply:
Kullanma konusunda karasız olduğum için yorum satırına ekledim